app/retail/emails.py

```diff
@@ -471,44 +471,42 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from external_bounties.models import ExternalBounty
-    subject = "iOS App Launches | Avatar Builder Launches "
+    subject = "Getting Started With Gitcoin | Gitcoin Profiles"
 
     intro = '''
 
 <p>
     Hi there
 </p>
 <p>
-This week, our subject line can't capture everything we shipped! On Monday, we released
- <a href="https://medium.com/gitcoin/gitcoin-ios-app-launches-3c5722170c4d">our iOS app!</a> You can now browse Gitcoin issues anywhere, anytime and
- express interest in working ones which fit your skill sets. We're very excited about the opportunity to 'Grow Open Source' and see how to contribute
- to leading projects without needing to be at your computer.
-</p>
-We also released <a href="https://medium.com/gitcoin/new-onboarding-flow-build-your-own-gitcoin-avatar-7b7a656955f9">Gitcoin Avatar Builder!</a> The Avatar Builder
-allows every Gitcoin user to customize their profile to their preferences, while also adding details on their skill sets and issues which interest them. We're excited
-to see everyone's beautiful (Avatar) faces on <a href="https://gitcoin.co/leaderboard">Gitcoin's Leaderboard</a> soon.
-<p>
-We've begun deploying funds from the <a href="https://medium.com/ecf-review/announcing-the-ecf-web-3-0-infrastructure-fund-pilot-program-ab8894af35fa">ECF Web 3.0 Infrastructure Fund pilot program.</a>
-It's an exciting addition to our MakerDAO + Ethereum Foundation grants. In tandem, $40,000 has been dedicated towards open source, and we're just at the beginning!
+This week, we shipped <a href="https://medium.com/gitcoin/getting-started-with-gitcoin-fa7149f2461a">
+Getting Started With Gitcoin.</a>
+In this post designed for our Gitcoin contributors, you’ll find all the info you
+need to set up your Gitcoin account and get to work contributing to open source projects!
 </p>
+We also published our first installment of
```

Comment:
@SaptakS i think next time this should be in a `<p>` tag http://bits.owocki.com/3L2J3b19462L/Screen%20Shot%202018-06-29%20at%203.48.42%20PM.png

Reply:
Will keep this in mind next time.